src/components/Nav/Menu/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useRef } from "react"
+import { Box, type BoxProps, Flex, Grid } from "@chakra-ui/react"
+import * as NavigationMenu from "@radix-ui/react-navigation-menu"
+import * as Portal from "@radix-ui/react-portal"
+
+import { Button } from "@/components/Buttons"
+
+import { SECTION_LABELS } from "@/lib/constants"
+
+import type { LvlRefs, NavSections } from "../types"
+
+import LvlContent from "./LvlContent"
+
+import { useRtlFlip } from "@/hooks/useRtlFlip"
+
+type NavMenuProps = BoxProps & {
+  sections: NavSections
+}
+
+const Menu = ({ sections, ...props }: NavMenuProps) => {
+  const { direction } = useRtlFlip()
+
+  const refs: LvlRefs = {
+    lvl1: useRef(null),
+    lvl2: useRef(null),
+    lvl3: useRef(null),
+  }
+
+  return (
+    <>
+      <Box {...props}>
+        <NavigationMenu.Root
+          dir={direction}
+          orientation="horizontal"
+          delayDuration={750}
+        >
+          <NavigationMenu.List asChild>
+            <Flex listStyleType="none">
+              {SECTION_LABELS.map((sectionKey) => {
+                const { label, items } = sections[sectionKey]
+                return (
+                  <NavigationMenu.Item key={sectionKey}>
+                    <NavigationMenu.Trigger asChild>
+                      <Button
+                        py="2"
+                        px="4"
+                        variant="ghost"
+                        color="body.base"
+                        sx={{
+                          '&[data-state="open"]': {
+                            bg: "primary.lowContrast",
+                            color: "primary.base",
+                          },
+                        }}
+                      >
+                        {label}
+                      </Button>
+                    </NavigationMenu.Trigger>
+                    <LvlContent lvl={1} items={items} refs={refs} />
+                  </NavigationMenu.Item>
+                )
+              })}
+            </Flex>
+          </NavigationMenu.List>
+          <Portal.Root container={refs.lvl1.current}>
+            <NavigationMenu.Viewport />
+          </Portal.Root>
+        </NavigationMenu.Root>
+      </Box>
+
+      <Grid
+        position="absolute"
+        visibility="hidden"
```

Comment:
Not blocker. Wondering if we should add some `aria-hidden` attribute here for sr, and switch it when the menu is shown.